ts_develop/conda/meta.yaml

```diff
@@ -1,6 +1,9 @@
 package:
   name: ts-develop
-  version: 0.3.19
+  version: 0.3.20
```

Comment:
Same as in ts-conda-build, you probably don't need to change the version

Reply:
Agreed, reverted!